modules/opengl/src/shader/shaderutils.cpp

```diff
@@ -481,11 +481,11 @@ void addShaderDefines(Shader& shader, const IsoTFProperty& property) {
 }
 
 void setShaderUniforms(Shader& shader, const IsoTFProperty& property) {
-    setShaderUniforms(shader, property.isovalues_, property.isovalues_.getIdentifier());
+    setShaderUniforms(shader, property.isovalues_);
 }
 
-void setShaderUniforms(Shader& shader, const IsoTFProperty& property, const std::string& name) {
-    setShaderUniforms(shader, property.isovalues_, name);
+void setShaderUniforms(Shader& shader, const IsoTFProperty& property, const std::string&) {
+    setShaderUniforms(shader, property.isovalues_, property.isovalues_.getIdentifier());
```

Comment:
This seems incorrect. Name should be used?

Reply:
you have take that up with @martinfalk, Im not touching that again.

Reply:
So this resolves a bug that was there before? The new code does not behave the same as the old

Reply:
the new code should be the same as the old old code...

Reply:
Hmm, that is weird. When I look at the last commit is says that 
setShaderUniforms(shader, property.isovalues_, name);
is changed to 
setShaderUniforms(shader, property.isovalues_, property.isovalues_.getIdentifier());

In the overall commit it does not. I guess you made that change and then back again?

Reply:
yes... I did not want to make any breaking changes....